examples/github/main.tf

```diff
@@ -3,20 +3,21 @@ terraform {
 
   required_providers {
     github = {
-      source  = "hashicorp/github"
-      version = ">= 4.1.0"
+      source  = "integrations/github"
+      # 4.3.1 is broken for personal account users
+      version = "4.3.0"
     }
     kubernetes = {
       source  = "hashicorp/kubernetes"
-      version = ">= 1.13.3"
+      version = ">= 2.0.1"
     }
     kubectl = {
       source  = "gavinbunney/kubectl"
       version = ">= 1.9.1"
     }
     flux = {
       source  = "fluxcd/flux"
-      version = ">= 0.0.1"
+      version = ">= 0.0.10"
```

Comment:
Would it be possible to change the version during release, making sure it's always the latest stable in the docs?

Reply:
Sounds like a good idea but we should add that in another PR.